### PR TITLE
build(app): update project core packages

### PR DESCRIPTION
**Pull request by vitorhfs**
https://app.asana.com/0/1202389996791452/1204324075522961/f

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    {
+      name: 'webkit',
+      use: { ...devices['Desktop Safari'] },
+    },
+
+    /* Test against mobile viewports. */
+    // {
+    //   name: 'Mobile Chrome',
+    //   use: { ...devices['Pixel 5'] },
+    // },
```
Comment (vitorhfs) [inline playwright.config.ts:54]: I think we should have this uncommented to have by default at least one mobile browser testing

File: e2e/example.spec.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@playwright/test';
```
Comment (vitorhfs) [inline e2e/example.spec.ts:1]: That's an extraneous dependency and the eslint throws an error on dev mode, it doesn't affect anything to this code, but to keep the culture of an error-free environment we should add a `// eslint-disable-next-line import/no-extraneous-dependencies` to this file and on `playwright.config` as well

**Review by vitorhfs — COMMENTED** [review]
That's a really great PR! Thanks for taking this over and leading the task so well 😄 

There are only small things I would like to request to make it shine brighter:
- Environment files and `polyfills.ts` needs to be indented correctly;

**Comment by vitorhfs** [comment]
There is also another error on `eslintrc.js` file. It is related to this issue: https://typescript-eslint.io/linting/troubleshooting/#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file

It tells us that we are trying to apply configurations to files that are not included on TS configuration, access the link to see the fix for this issue, if takes you too long to check, please ignore this error for now

File: playwright.config.ts (hunk 1 above)
Comment (lfa-vasconcellos) [reply playwright.config.ts:55]: Not really, this file will invariably need to be changed on every project. So there's no point in prematurely configuring this.

Comment (vitorhfs) [reply playwright.config.ts:55]: Fair enough

**Review by lfa-vasconcellos — APPROVED** [review]

**Pull request merged**